atlas/getting-started/deploy-from-blueprint.mdx

```diff
@@ -22,37 +22,32 @@ After Atlas connects to your GitHub account, the `Connect` button indicates that
 
 ![Connected to GitHub](./assets/img/connect-to-github-blueprints.png)
 
-### 2. Clone the blueprint and choose the repository
+### 2. Create a repository
 
-Now that GitHub is connected, it's time to create a repository by cloning a blueprint.
+Now that GitHub is connected, it's time to create a repository where your Blueprint code is going to be stored.
 
-1. Select `Clone Blueprint`. This option opens another tab in GitHub.
+1. Select your GitHub account
+<img width="559" alt="image" src="https://user-images.githubusercontent.com/15729608/177573589-ed412e4c-e849-4f5c-a281-4de9c424932d.png">
 
-2. On the `Create a new repository` page, set your new git repository settings.
+In case you don't see the account for which you'd like to create a repository on the list, click on "Manage GitHub app" on the top of the screen:
+<img width="602" alt="image" src="https://user-images.githubusercontent.com/15729608/177573864-91d5a0f6-ba7e-4d27-9a04-75182d548627.png">
 
-- Under `Owner`, select the username you associated with your repository.
-- For `Repository name`, enter a name in a format similar to `repository-name`. Avoid capitalization or spaces to make it easy to navigate your repository in the future.
-- (Optional) Enter a `description` for your repository.
-- Choose whether to make your repository `Public` or `Private`.
-- Leave `Include all branches` unchecked.
 
-3. With your new repository named, select `Create repository from template`.
+2. Type a name of your repository
 
-![Create a new repo](./assets/img/new-repo-screen.png)
+Depending on which Blueprint you chose, Atlas will suggest a repository name for you. Name of your repository should be unique within your GitHub account - in case it's not, you'll be returned an error after clicking the "Create" button on the bottom of the page.
 
-4. Navigate back to the WP Engine User Portal by closing the GitHub tab.
+3. Select the visibility of your repository
 
-5. Select your new repository from the repository list when it refreshes.
+If you want your repository to be visible to people that can only access your GitHub account, make sure to check the "Private repository" checkbox.
 
-![Select your repo](./assets/img/repo-select.png)
-
-6. Select a region for your Atlas app. The region you select will be used for all of your app's environments (production/staging/etc.) and cannot be changed once your Atlas application has been deployed.
+4. Select a region for your Atlas app. The region you select will be used for all of your app's environments (production/staging/etc.) and cannot be changed once your Atlas application has been deployed.
 
 ![Select your app's region](./assets/img/region-select-blueprint.png)
 
 7. Click `Create App`.
 
-That's it! Track the initial build progress on the next screen that loads automatically.
+That's it! Track the initial build progress on the next screen that loads automatically. The repository with Blueprint code as well as new WordPress install 
```

Comment:
I'm not sure we need this last sentence. Are we describing what they should see on the next screen, or just emphasizing that the BP is frontend code as well as WP config?